pkg/router/router.go

```diff
@@ -351,9 +351,9 @@ func (r *router) serveSetup() {
 			log := r.logger.WithError(err)
 			if err == dmsg.ErrEntityClosed {
```

Comment:
for the sake of consistency I guess, this one matches the error by value, the one on the file below (transport/manager.go) uses proper error type checks (`errors.Is`), probably use that also?